pkg/geo/geomfn/affine_transforms_test.go

```diff
@@ -530,15 +523,8 @@ func TestRotateWithPointOrigin(t *testing.T) {
 				require.EqualError(t, err, tt.wantErrStr)
 			} else {
 				require.NoError(t, err)
+				geometryInEpsilon(t, geometryFromGeomT(t,tt.wantGeom), got, 1e-5)
```

Comment:
should this be in the same scope as the previous code (i.e. outside the else block)?

Reply:
this one was deliberate -- we should never assert the value if an error occurs (unless it was part of the contract)